demos/sports-analytics-architecture/main.bicep

```diff
@@ -14,18 +14,18 @@ Name of the Azure Data Lake Storage Gen2 storage account. Storage account name r
 ''')
 @minLength(3)
 @maxLength(24)
-param azureDataLakeStoreAccountName string = 'stor${uniqueString(resourceGroup().id)}'
```

Comment:
Let's leave the defaultValues in so it's easier to deploy...

Reply:
Updated.